packages/store/tests/state.spec.ts

```diff
@@ -163,4 +170,84 @@ describe('State', () => {
       expect(TestBed.get(Store).snapshot().foo).toEqual(['updateState', 'onInit']);
     });
   });
+
+  describe('"ngxsAfterBootstrap" lifecycle hook', () => {
+    function createRootNode(selector = 'app-root'): void {
+      const document = TestBed.get(DOCUMENT);
+      const adapter: DomAdapter = new BrowserDomAdapter();
+
+      const root = adapter.firstChild(
+        adapter.content(adapter.createTemplate(`<${selector}></${selector}>`))
+      );
+
+      const oldRoots = adapter.querySelectorAll(document, selector);
+      oldRoots.forEach(oldRoot => adapter.remove(oldRoot));
+
+      adapter.appendChild(document.body, root);
+    }
+
+    const enum LifecycleHooks {
+      NgOnInit = 'ngOnInit',
+      NgxsOnInit = 'ngxsOnInit',
+      NgAfterViewInit = 'ngAfterViewInit',
+      NgxsAfterBootstrap = 'ngxsAfterBootstrap'
+    }
+
+    let hooks: LifecycleHooks[] = [];
+
+    @Component({
+      selector: 'app-root',
+      template: ''
+    })
+    class MockComponent implements OnInit, AfterViewInit {
+      public ngOnInit(): void {
+        hooks.push(LifecycleHooks.NgOnInit);
+      }
+
+      public ngAfterViewInit(): void {
+        hooks.push(LifecycleHooks.NgAfterViewInit);
+      }
+    }
+
+    function createModule() {
+      @NgModule({
+        imports: [BrowserModule],
+        declarations: [MockComponent],
+        entryComponents: [MockComponent]
+      })
+      class MockModule {}
+
+      return MockModule;
+    }
+
+    beforeEach(() => (hooks = []));
+
+    it('should invoke "ngxsAfterBootstrap" after "ngxsOnInit" and after root component\'s "ngAfterViewInit"', () => {
+      @State({ name: 'foo' })
+      class FooState implements NgxsOnInit, NgxsAfterBootstrap {
+        public ngxsOnInit(): void {
+          hooks.push(LifecycleHooks.NgxsOnInit);
+        }
+
+        public ngxsAfterBootstrap(): void {
+          hooks.push(LifecycleHooks.NgxsAfterBootstrap);
+        }
+      }
+
+      TestBed.configureTestingModule({
+        imports: [createModule(), NgxsModule.forRoot([FooState])]
+      });
+
+      const app: ApplicationRef = TestBed.get(ApplicationRef);
```

Comment:
```ts
MockModule implements NgDoBootstrap {
 public static ngDoBootstrap(app: ApplicationRef) {
   createRootNode();
   app.bootstrap(MockComponent);
 }
}
```

```ts
MockModule.ngDoBoostrap(TestBed.get(ApplicationRef))
```

Reply:
Does this lifecyle hook get called for lazy loaded feature modules?